cross/sickchill/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME = SickChill
+PKG_VERS = 2020.11.24-1
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/SickChill/SickChill/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+SC_GIT_BRANCH=master
+SC_GIT_COMMIT_HASH=46a7f2a2304d6abbc6459820f88d830c57e89bcb
+SC_GIT_COMMIT_BRANCH=v$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = https://sickchill.github.io/
+COMMENT = Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.
+LICENSE = GPL
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = nop
+INSTALL_TARGET = sickchill_install
+
+include ../../mk/spksrc.cross-cc.mk
```

Comment:
So if I understand well, there is no source file to cross-compile here, so it does not make sense to have "cross/sickchill".
As a pip package is available, I would directly grab it from `spk/sickchill/src/requirements.txt` with an explicit tagged version (please do not get "master") https://github.com/SickChill/SickChill/wiki/Pip
If you need file structure in package, you may prepare them in `spk/sickchill/Makefile` as install target

Reply:
According to miigotu, using the pip package would render further updating manual only (see: https://github.com/SynoCommunity/spksrc/pull/4161#issuecomment-706717034 ), it seems it was ruled out at that time.

Reply:
I’ve read through all the comments and it seems that this is indeed the case. What about the configure/compile though, @Nyaran

Reply:
Does that mean there will be no Synocommunity package for auto-updatable Sickchill ? :-(

Reply:
As long as nobody is taking care of this package, PR and the discussion, nothing at all will happen...

Reply:
Sorry, I was off for a while...

So, now the only issue is to remove the "cross/compile" part, isn't it? I will try it